samcli/commands/validate/validate.py

```diff
@@ -95,3 +114,40 @@ def _read_sam_file(template):
         sam_template = yaml_parse(sam_template.read())
 
     return sam_template
+
+def _lint(ctx, template):
+    """
+    Parses provided SAM template and maps errors from CloudFormation template back to SAM template.
+
+    Parameters
+    -----------
+    ctx
+        Click context object
+    template
+        Path to the template file
+
+    """
+
+    try:
+        if ctx.debug:
+            (args, filenames, formatter) = cfnlint.core.get_args_filenames([template, "--debug"])
+        else:
+            (args, filenames, formatter) = cfnlint.core.get_args_filenames([template])
+        LOGGER.setLevel(logging.WARNING)
```

Comment:
Why we need to explicitly set the logging level here?

Reply:
Similar issue to logging issue mentioned above. Without the cfnlint logger being explicitly set to a higher level, a lot of INFO level logs will be exposed when they shouldnt be. Again, changing the base logging level in main.py could solve this but not sure what side effects that would have across sam cli.